modules/labs-react/pagination/lib/Pagination/common/utils/space.ts

```diff
@@ -1,6 +1,6 @@
-import {spacing} from '@workday/canvas-kit-react/core';
+import {space as canvasSpace} from '@workday/canvas-kit-react/core';
 
-export type Spacing = keyof typeof spacing;
+export type Spacing = keyof typeof canvasSpace;
```

Comment:
This update was all manual. This is a placeholder and will be replaced when the Layout components are in CK.